src/Modules/Backend/assets/Backend/webpack/scss/components/_loader.scss

```diff
@@ -1,4 +1,4 @@
-#ajaxSpinner {
+#ajax-spinner {
```

Comment:
@carakas I have a task for @daphneslootmans to cleanup CSS and JS when fork6 is a bit further. 
Let's focus here on the CI issue, otherwise this PR will get out of focus.